src/Requestor.js

```diff
@@ -63,7 +56,19 @@ export default function Requestor(platform, options, environment) {
             const message = messages.invalidContentType(result.header('content-type') || '');
             return Promise.reject(new errors.ULFlagFetchError(message));
           }
-        } else {
+        }
+        else if (result.status === 400) {
```

Comment:
I want to test this change but don't know how. I tried npm link, and run a test script using node but I got error at this line `if (window.XMLHttpRequest) {` in ulClient.js. The reason is that `window` is undefined in for node.

Reply:
You'd have to test this in browser. I created an index.html for it. Let me find and ping you on Slack.